src/components/ImageSlider/ImageScrollView.web.js

```diff
@@ -16,6 +16,20 @@ const ImageScrollView = React.forwardRef((props, ref) => {
     onClickWeb,
   } = props
 
+  if (images.length === 1) {
```

Comment:
Will images always be an array? Is it possible that it will be `undefined` or `null`?

Reply:
You're totally right, I'll fix that

Reply:
Actually, never mind. Code inside `ImageList.js` checks for that and this code won't run if `images` is undefined or isn't an array.